### PR TITLE
Added some basic contributing guidelines

### PR DESCRIPTION
**Pull request by zwoop**

### CODE REVIEW

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/freebsd-github/1547/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/linux-github/1441/ for details.

**Comment by atsci** [comment]
clang-analyzer build *successful*! See https://ci.trafficserver.apache.org/job/clang-analyzer-github/112/ for details.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+3. If there is an issue/feature and code, then create a Github Pull Request.
+   If there is an existing Jira Ticket or Github Issue refer to the Ticket or
+   Issue in the Pull Request.  Creating a Github Issue is not required for a
+   Github Pull request. 
+
+
+Making a good PR
+----------------
+
+Since Github _PRs_ and _Issues- are now our primary way of tracking both code
```
Comment (bryansou) [inline CONTRIBUTING.md]: `_Issues-` should be `_Issues_`

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/freebsd-github/1553/ for details.

File: CONTRIBUTING.md (hunk 1 above)
Comment (zwoop) [inline CONTRIBUTING.md]: Thanks!

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/linux-github/1447/ for details.

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/freebsd-github/1554/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/linux-github/1448/ for details.

**Comment by atsci** [comment]
clang-analyzer build *successful*! See https://ci.trafficserver.apache.org/job/clang-analyzer-github/118/ for details.

**Comment by atsci** [comment]
clang-analyzer build *successful*! See https://ci.trafficserver.apache.org/job/clang-analyzer-github/119/ for details.

**Pull request merged**